drivers/clock_control/Kconfig.stm32

```diff
@@ -176,6 +176,49 @@ config CLOCK_STM32_PLL2_PREDIV2
 
 endif # SOC_SERIES_STM32F1X
 
+if SOC_SERIES_STM32F2X
+
+config CLOCK_STM32_PLL_M_DIVISOR
+	int "Division factor for PLL VCO input clock"
+	depends on CLOCK_STM32_SYSCLK_SRC_PLL
+	default 20
+	range 2 63
+	help
+	 PLLM division factor needs to be set correctly to ensure that the VCO
+	 input frequency ranges from 1 to 2 MHz. It is recommended to select a
+	 frequency of 2 MHz to limit PLL jitter.
+	 Allowed values: 2-63
+
+config CLOCK_STM32_PLL_N_MULTIPLIER
+	int "Multiplier factor for PLL VCO output clock"
+	depends on CLOCK_STM32_SYSCLK_SRC_PLL
+	default 192
+	range 192 432
```

Comment:
According to reference manual range is 64 > 432.

Reply:
based on RM0033 Page 96, PLLN range is 192 ~ 432.

Reply:
So, from RM0033, I can read, page 98:
"VCO output frequency = VCO input frequency × PLLN with 64 ≤ PLLN ≤ 432"

Reply:
PLLN range is different between RN0033 version 7 and version8.

Version 7: VCO output frequency = VCO input frequency × PLLN with 192 £ PLLN £ 432
Version 8: VCO output frequency = VCO input frequency × PLLN with 64 ≤ PLLN ≤ 432

Beside the PLLN can be select in cubemx is 192 ~ 432.

Reply:
The attachment contains the information regarding the PLLN value range collected from the RM0033 Rev2 December 2010 to
RM0033 Rev8 Mai 2018
and
STM32Cube_FW_F2_V1.1.0 V1.1.1 30-December-2014 to
STM32Cube_FW_F2_V1.7.0 V1.7.0 / 29-September-2017

1. RM0033 REV2 December 2010 to RM0033 REV3 April 2011:
    192 <= PLLN <= 432 with VCO output frequency is between 192 and 432 MHz
2. RM0033 REV4 December 2011 to RM0033 REV6 September 2013:
     64 <= PLLN <= 432 with VCO output frequency is between 64 and 432 MHz
3. RM0033 REV7 February 2015:
   192 <=  PLLN <=  432  with VCO output frequency is between 192 and 432 MHz
4. RM0033 REV8 Mai 2018:
     64 <= PLLN <= 432 with VCO output frequency is between 64 and 432 MHz
5. All FW from STM32Cube_FW_F2_V1.1.0 V1.1.1 30-December-2014 to
         STM32Cube_FW_F2_V1.7.0 V1.7.0 / 29-September-2017
   192 <=  PLLN <=  432  with VCO output frequency is between 192 and 432 MHz
6. STM32Cube
   192 <=  PLLN <=  432

My advice is to keep the 192 to 432 range until a new FW release appears reflecting the latest changes in RM0033 REV8 Mai 2018.

[RCC PLL configuration register (RCC_PLLCFGR) PLLN value range.zip](https://github.com/zephyrproject-rtos/zephyr/files/2108263/RCC.PLL.configuration.register.RCC_PLLCFGR.PLLN.value.range.zip)

Reply:
Ok, I confirm that 192 is the good value.
Reference manual will be updated with this value